app/controllers/estimates_controller.rb

```diff
@@ -0,0 +1,23 @@
+
+class EstimatesController < ApplicationController
+  require 'active_shipping'
+  require "./lib/estimator/estimator.rb"
+
+  def quote
+    ship_params = strong_shipping_params
+    estimate = Estimator::Estimate.query(ship_params)
+    if estimate
+      render :json => estimate.to_json, :status => :ok
+    else
+      render :json => [], :status => :no_content
+    end
+  end
```

Comment:
I <3 <3 <3 this `quote` method. It's only doing one thing and is super easy to follow.